format/spec.md

```diff
@@ -1861,6 +1861,16 @@ Java writes `-1` for "no current snapshot" with V1 and V2 tables and considers t
 
 Some implementations require that GZIP compressed files have the suffix `.gz.metadata.json` to be read correctly. The Java reference implementation can additionally read GZIP compressed files with the suffix `metadata.json.gz`.  
 
+### Schema evolution and writing with old schemas
+
+Writers must write out all fields with the types specified from the table schema. Writers should use the latest schema for writing. Not writing out all columns or not using the latest schema can change the semantics data written. The following are possible inconsistencies that can be introduced:
```

Comment:
[minor] 
> all fields with the types specified from the table schema

is it worth explicitly clarifying that this can be any schema as saying table schema might mean latest schema, from my understanding the intention is MUST is only for writing all the columns, using latest schema is good to have.

Reply:
Thank you @emkornfield ! recent rev looks great !